
# Author : BEN SWISA
node1:
  ros__parameters:
    enc_y : 1
    enc_z : 0
    max_pwm : 200 # upper bound for motor pwm
    controller_freq : 100
    manual_control : 0 # 1 if you would like to control the engines manualy (send pwm signals manually)
    manual_motor_pwm:  [0,0,0,
                        0,0,0,
                        0,0,0,
                        0,0,0] # if sending manual pwm outputs this is the array of the signalls being sent
    kp : [12.0000000,100.000000,10.0000000,
          12.0000000,100.000000,10.0000000,
          0.0000000,0.0000000,0.0000000,
          0.0000000,0.0000000,0.0000000] # controller constants
    ki : [0.00003010,0.0006000,0.00003010,
          0.00003010,0.0006000,0.00003010,
          0.0000000,0.0000000,0.0000000,
          0.0000000,0.0000000,0.0000000]  
    kd : [2.000000,50.000000,10.0000000,
          2.0000000,50.000000,2.0000000,
          0.0000000,0.0000000,0.0000000,
          0.0000000,0.0000000,0.0000000]

#     kp : [0.0000000,0.000000,0.0000000,
#           12.0000000,100.000000,10.0000000,
#           0.0000000,0.0000000,0.0000000,
#           0.0000000,0.0000000,0.0000000] # controller constants
#     ki : [0.0000000,0.000000,0.0000000,
#           0.00003010,0.0006000,0.00003010,
#           0.0000000,0.0000000,0.0000000,
#           0.0000000,0.0000000,0.0000000]  
#     kd : [0.0000000,0.000000,0.0000000,
#           2.000000,50.000000,2.0000000,
#           0.0000000,0.0000000,0.0000000,
#           0.0000000,0.0000000,0.0000000]

node2:
  ros__parameters:
    joint_angle_cmd :  [0.000000,0.000000,
                        0.0000000,0.0000000,
                        0.0000000,0.0000000]
